src/main/java/org/abstractj/kalium/NaCl.java

```diff
@@ -24,105 +24,130 @@
 
 public class NaCl {
 
```

Comment:
@vshaipov I think these changes here are related with trailing spaces and not relevant to the project, probably caused by the IDE configuration. 

Could you please fix this?